### PR TITLE
Release/hull hubspot accounts

### PR DESCRIPTION
**Pull request by michaloo**

### CODE REVIEW

**Comment by michaloo** [comment]
@timliuhull ha, I was curious what would you do if I request a review from you :)
but definitely let's do a walkthrough together on a call.

File: packages/hull/src/handlers/schedule-handler/factory.js [hunk 1]
```diff
@@ -80,14 +80,14 @@ function scheduleHandlerFactory(
 
     if (fireAndForget === true) {
       callbackResult.catch(error => {
+        // all TransientErrors (and child error classes such as ConfigurationError)
         if (error instanceof TransientError) {
```
Comment (timliuhull) [inline packages/hull/src/handlers/schedule-handler/factory.js]: doesn't this need to be (error instanceof TransientError || error instanceof ConfigurationError)

Comment (michaloo) [reply packages/hull/src/handlers/schedule-handler/factory.js]: let me respond in javascript:
```
class ParentClass {}
class ChildClass extends ParentClass {}
const testInstance = new ChildClass();

(testInstance instanceof ParentClass) === true;
(testInstance instanceof ChildClass) === true;
```

Comment (timliuhull) [reply packages/hull/src/handlers/schedule-handler/factory.js]: ah, i didn't know ConfigurationError extended TransientError

Comment (michaloo) [reply packages/hull/src/handlers/schedule-handler/factory.js]: In my understanding it's kind of transient error which can be gone after `ship:update` (aka configuration change), what do you think?

**Pull request merged**